utils/exceptions.hh

```diff
@@ -21,6 +21,9 @@
 
 #pragma once
 
+#include <seastar/core/sstring.hh>
+#include <seastar/util/log.hh>
+
```

Comment:
Please use a forward declaration for seastar::logger to avoid compile-time bloat.